### ISSUE TITLE
Some processes that can be improved in Otter dev work

### ISSUE THREAD

**Issue by arinaturcu** [Codeinwp/otter-blocks#1650]
Some processes in the Otter dev work that can be improved (for my rock for this Quarter).

## 1. Php standard
PHPCS does not work with PHP 7.4 or later. Many times we push the code and then fix the PHPCS errors after we see them in the PR checks. 

## 2. Bem helper
We could start adding CSS classes with [BEM](https://getbem.com/) in mind. It will make the CSS files look better and maybe make the overall CSS size smaller.

A nice touch here would be to implement a BEM helper that will generate class names, something like [react-bem-helper
](https://www.npmjs.com/package/react-bem-helper) or even use that package, but implementing it should be fairly easy.

## 3. PHP tests
Some additional PHP tests might be helpful.

## 4. Otter child plugins usage
Right now, when developing the Otter repo, we can build and use only the entire Otter plugin. So, if we need for example just the `Block CSS` plugin, we need to take it separately and move it to another instance, which takes unnecessary time. Maybe we can improve this.

Feedback is appreciated @Codeinwp/otter-team.

**Comment by HardeepAsrani** [comment]
I agree with the 1 & 4. These seem very good. BEM is good too just that we can only implement it in new blocks and not the old ones.
